server/config/config.go

```diff
@@ -752,6 +754,10 @@ type ScheduleConfig struct {
 
 	// The day of hot regions data to be reserved. 0 means close.
 	HotRegionsReservedDays uint64 `toml:"hot-regions-reserved-days" json:"hot-regions-reserved-days"`
+
+	// The threshold of region size for balance hot region scheduler and split bucket.
+	// It will be split for move hot region if region is bigger than HotRegionSplitSize.
+	HotRegionSplitSize int64 `toml:"hot-region-split-size" json:"hot-region-split-size,omitempty"`
```

Comment:
maybe `min-moveable-hot-peer-size` more suitable?

Reply:
it will rename as `max-movable-hot-peer-size`.